free-courses-en.md

```diff
@@ -210,6 +217,7 @@
 
 ### Ruby
 
+* [CodeSchool - Getting Started With Ruby on Rails](https://www.codeschool.com/learn/ruby)
```

Comment:
I see a "free" label on the first two chapters. Is this thing really free, or only the first two chapters?